src/shared/default-routes.js

```diff
@@ -40,16 +41,16 @@ const DefaultRoutes = ({
       }
       <Route
         path='/'
-        component={FrontPage || Page}  />
+        component={fetchData(FrontPage || Page || Error)}  />
       <Route
         path=':page(/:subpage)'
-        component={Page} />
+        component={fetchData(Page || Error)} />
```

Comment:
Yep they'd not see it anymore. I was assuming this was like `fallback` prop in the previous setup. https://github.com/shortlist-digital/tapestry-wp/blob/develop/src/shared/default-routes.js#L23 I can take it out.